### ISSUE TITLE
Add Glob type import

### ISSUE THREAD

**Issue by Prezto** [mholt/caddy#219]
When i wanted to setup Caddy with systemd as a service i noticed that the import gives me the following error when i use "import /etc/caddy/conf.d/" from the file "etc/caddy/caddy.conf".

"Aug 12 15:24:13 localhost.localdomain caddy[24058]: caddy.conf:1 - Parse error: Could not import /etc/caddy/conf.d/* - open /etc/caddy/conf.d/*: no such file or directory"

Maybe this would be a useful feature to add.

Test system info :
 - Linux localhost.localdomain 3.10.0-229.el7.x86_64 #1 SMP Fri Mar 6 11:36:42 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
 - Caddy 0.7.5 with Git

**Comment by mholt** [comment]
This is a possibility. Is it OK if ordering doesn't matter? (In reality, they would probably be lexically ordered but I'm not 100% sure that's reliable cross-platform.)

**Comment by Prezto** [comment]
I think ordering does not matter for the server blocks. i cloned the tree to see if i can find a solution in the code myself. It will be my first time with Go.

**Comment by mholt** [comment]
Cool! Let us know if you have any questions. You can also find us on Slack if you need some pointers (see the contributing file).

**Comment by Prezto** [comment]
Just an FYI. I have been busy with my daily work. So i did not find time and will not have time for at least a week. But i will try to contribute.

**Comment by mholt** [comment]
Ping. :smile: How's this going @Prezto?

**Comment by Prezto** [comment]
Hello @mholt. I did work on something, but i am learning Go and this the first thing i do in this language after python, Ruby and Php. So it is difficult for me.

Nonetheless i have uploaded what i have tried to work on in a repo here on github: https://github.com/Prezto/import/tree/master

Maybe i can get some assistance on this one?

**Comment by mholt** [comment]
Sure, no problem.

First, good start. You have the right idea with the Glob function and looping over the files. But this doesn't have to be an add-on; in fact, it shouldn't be. You can build this [right into the parser](https://github.com/mholt/caddy/blob/3614a093e3cfddaa96e638a808d5a473bd5410f7/config/parse/parsing.go#L182).

Before continuing, I recommend running `go fmt` on all your Go code to clean up the indentation a bit. Also use `go build` to fix syntax and other compile-time errors with your code.

**Comment by Prezto** [comment]
Thank you for the advice. I will do it in the parser and clean things up this week.

Verzonden met mijn Windows Phone
________________________________
Van: Matt Holt<mailto:notifications@github.com>
Verzonden: ‎19-‎10-‎2015 03:33
Aan: mholt/caddy<mailto:caddy@noreply.github.com>
CC: Dany Henriquez<mailto:dany.henriquez@outlook.com>
Onderwerp: Re: [caddy] Add Glob type import (#219)

Sure, no problem.

First, good start. You have the right idea with the Glob function and looping over the files. But this doesn't have to be an add-on; in fact, it shouldn't be. You can build this [right into the parser](https://github.com/mholt/caddy/blob/3614a093e3cfddaa96e638a808d5a473bd5410f7/config/parse/parsing.go#L182).

Before continuing, I recommend running `go fmt` on all your Go code to clean up the indentation a bit. Also use `go build` to fix syntax and other compile-time errors with your code.

---

**Comment by Prezto** [comment]
I tried, but I need this now and I cannot get it to work. This is because I am new to go-Lang. So I solved it in a different way by concatenating with bash before starting caddy with systemd.

I tried, but failed and will try to learn more before I have another shot at this  :)

**Comment by mholt** [comment]
It's no problem at all! We'll mark this as deferred so we don't forget about it. Quite often, we have contributors looking for something to do, so now we can point them to this.

**Comment by mholt** [comment]
FWIW @Prezto this was implemented almost a year ago but I forgot to ping you. 😛
